tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -30,22 +30,31 @@ import org.apache.spark.sql.types.DataTypes
 
 class RegularExpressionTranspilerSuite extends FunSuite with Arm {
 
+  test("") {
```

Comment:
Odd we don't even name this test?

Reply:
Oops. I was working on this late last night and thought I had already pushed the fix for that.